public/docs/ts/latest/guide/index.jade

```diff
@@ -1,109 +1,111 @@
+  example from where QuickStart leaves off. It may appear simple, but it has the essential characteristics of a professional application such as
+  a sensible project structure, data binding, master/detail, services, dependency injection, navigation, and remote data access.
+
+    The final iteration of the "[Tour of Heroes](#toh)" is a positive answer to that most important question:
   ***can we build an Angular 2 application that does what we need it to do?***
 
     Return from the tutorial to the basic chapters of this Developers Guide.
-    
+
   1. [Displaying Data](displaying-data.html) explains how to get information on to the screen.
-  
+
   1. [User Input](user-input.html) covers the basics of responding to user behavior.
-  
+
   1. [Forms](forms.html) handle user data entry and validation within the UI.
-  
+
   1. [Dependency Injection](dependency-injection.html) is the way we build large, maintainable applications
   from small, single purpose parts.
-  
-  1. [Template Syntax](template-syntax.html) is a comprehensive study of Angular template HTML. 
+
+  1. [Template Syntax](template-syntax.html) is a comprehensive study of Angular template HTML.
 
   With this foundation, we can read and understand any chapter in the guide.
- 
+
   ## Other Resources
-  Don't neglect the other documentation resources. 
-  
-  The "[Cheat Sheet](cheatsheet.html)" is a handy map to Angular overall.
-  
-  The [API Guide](../api/) is the authority on every public-facing member of the Angular libraries.
- 
- 
-<a id="toh"></a>  
+  There are other resources that are useful, particularly for referencing specific details.
+
+  1. The "[Cheat Sheet](cheatsheet.html)" has a quick list of the syntax for most of the primary use cases.
+
+  1. The [API Guide](../api/) is the authority on every public-facing member of the Angular libraries.
+
+
+<a id="toh"></a>
 .l-main-section
 :marked
-  # Appendix: The Hero Staffing Agency
-  The final phase of the tutorial Tour of Heroes app runs like this.
-figure.image-display
-  img(src='/resources/images/devguide/toh/toh-anim.gif' alt="Tour of Heroes in Action" height="300px")
-:marked
-  There's a backstory to the "Tour of Heroes" and every sample in this guide.
-  
-  The world is full of crises large and small. 
-  Fortunately, there are courageous heroes prepared to take on every challenge.
-  The shadowy "Hero Staffing Agency" matches crises to heroes.
-  
-  We are contract developers, hired by The Agency to build an Angular application to manage their operations.
-  The Agency maintains a stable of heroes with special powers.
-  Ordinary humans call the agency, looking for help with their personal or global crises.
-  The Agency triages the crises and turns them into job requests. 
-  The heroes on staff bid to take a job and The Agency
-  assigns each job accordingly. 
-  
-  Our application handles every detail of recruiting, tracking and job assignment.  
+  # About the examples: The Hero Staffing Agency
```

Comment:
I think the entire appendix is confusing and out of place. I intend to delete it.